src/pyodide/internal/python.ts

```diff
@@ -102,5 +102,18 @@ export async function loadPyodide(
   const pyodide = Module.API.public_api;
 
   finishSnapshotSetup(pyodide);
+
+  Module.API.initializeStreams();
+
+  // Need to set these here so that the logs go to the right context. If we don't they will go
+  // to SetupEmscripten's context and end up being KJ_LOG'd, which we do not want.
+  pyodide.setStdout({
+    batched: (msg: any) => console.log(msg),
+  });
+
+  pyodide.setStderr({
+    batched: (msg: any) => console.error(msg),
+  });
```

Comment:
```suggestion
  // Need to call this so that the logs go to the right context. If we don't they will go
  // to SetupEmscripten's context and end up being KJ_LOG'd, which we do not want.
  Module.API.initializeStreams(null, (msg) => console.log(msg), (msg) => console.error(msg));
```